lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1440,7 +1440,7 @@ def test_progress_queries_paced_courses(self, self_paced, query_count):
 
     @patch.dict(settings.FEATURES, {'ASSUME_ZERO_GRADE_IF_ABSENT_FOR_ALL_TESTS': False})
     @ddt.data(
-        (False, 45, 29),
+        (False, 45, 28),
```

Comment:
This set of changes is just because some tests count the sql queries that happen? Which is real brittle. But it changed because now we check is_issuance_enabled.